### ISSUE TITLE
Call to Arms: Faction Balance honor not added

### ISSUE THREAD

**Issue by nexumus** [SunstriderEmu/BugTracker#3456]
**Describe the bug**
Call to Arms quests give 419h without any addition from faction balance - 50% alliance / 25% horde

**To Reproduce**
Take a "Call to Arms" quest, whether wsg etc. win, turn over - you'll receive exactly 419honor instead of: 
628 for Alliance
524 for Horde

**Screenshots/videos**
![honor](https://user-images.githubusercontent.com/63930775/82559653-4a18c080-9b70-11ea-80ad-b3bdba91e18b.jpg)

Long Story short - i can make an entry for everything, flag capping, base burning etc. -> you don't get any additional honor for pvp objectives apart from killing other players.

This makes pvp (bg) totally pointless, it's more benefitial to actually farm other graveyard than do objectives.

**Comment by astrateslol** [comment]
Hello, is it going to be fixed?

**Comment by hufsa** [comment]
Fixed with next server update.
